powermon/__init__.py

```diff
@@ -47,17 +43,17 @@ class ConfigError(Exception):
 """
 
 
-def readYamlFile(yamlFile=None):
-    _yaml = {}
-    if yamlFile is not None:
+def readConfigFile(configFile=None):
```

Comment:
i renamed this intentionally

Reply:
You moved everything, it was a several hour merge effort. As a result some of your changes will be reverted.